src/test/test-errno-util.c

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "errno-util.h"
+#include "string-util.h"
+#include "tests.h"
+
+TEST(test_strerror_not_threadsafe) {
+        /* Just check that strerror really is not thread-safe. */
+        log_info("strerror(%d) → %s", 200, strerror(200));  /* lgtm [cpp/potentially-dangerous-function] */
```

Comment:
LGTM.com was dropped from our CI. I think the comment is meaningless and can be dropped. cc @mrc0mmand

Reply:
Yeah, the code annotations don't work with CodeQL, so you can drop them (you can dismiss the alerts on GH instead, which does the same thing).

Reply:
I added another patch to drop the rest of those annotations.